objects/rct2/station/rct2.station.abstract.json

```diff
@@ -17,7 +17,8 @@
     ],
     "strings": {
         "name": {
-            "en-GB": "Abstract"
+            "en-GB": "Abstract",
+	    "ko-KR": "추상적"
```

Comment:
Could you fix the alignment of this line? It's also off in some of the other files.

Reply:
Done. Sorry ;)